docs/motivation.md

```diff
@@ -0,0 +1,13 @@
+# Motivation
+
+The first practical application of this package (even before we refactor other automation repos into it) is to orchestrate the process of generating STAC Items for datasets which have already been built with Pangeo Forge. All of the metadata required for cataloging already exists _somewhere_ in Pangeo Forge:
```

Comment:
Maybe add a link for the words `STAC Items`, to help the uninitiated reader understand what you mean.